WalletWasabi.Fluent/Extensions/TransactionSummaryExtensions.cs

```diff
@@ -25,13 +25,8 @@ feeDisplayUnit switch
 			_ => throw new InvalidOperationException($"Invalid Fee Display Unit value: {feeDisplayUnit}")
 		};
 
-	public static string? ToFeeDisplayUnitString(this Money? fee)
+	public static string ToFeeDisplayUnitString(this Money fee)
```

Comment:
I think it would make sense to allow `null` as a parameter and return `Unkown` string if the parameter is null. That would fix the warning and also remove some duplications.

Reply:
To me it feels weird to call a method on/with a `null` value, but it works and is less change in the code, so it makes sense.